openlibrary/templates/interstitial.html

```diff
@@ -1,16 +1,20 @@
-$def with(url, provider="", wait=5)
+$def with(url, provider_name="", wait=5)
 
-<h1>$_("You are being redirected to your book")</h1>
+$if provider_name is None:
+  $ provider_name = ""
 
-<p id="countdown">$_("This book is provided by %s, a third-party Open Library Trusted Book Provider", "<strong>" + provider= "</strong>")</p>
+<div class="interstitial">
+  <h1>$_("You are being redirected to your book")</h1>
 
-<p>$_("You will be automatically redirected to %s in %s seconds...", url, '<span id="timer">5</span>')</p>
+  <p id="countdown">$:_('This book is provided by %(book_provider)s, a third-party Open Library Trusted Book Provider', book_provider='<strong>' + websafe(provider_name).replace('_', ' ').title() + '</strong>')</p>
 
-<p>
-  <a href="#" onclick="window.close(); return false;">$_("Cancel")</a> |
-  <a href="">$_("Continue without waiting")</a>
-</p>
+  <p>$:_('In %(time)s seconds, you will be automatically redirected to: %(url)s', time='<span id="timer">{}</span>'.format(wait), url='<a href="{0}">{0}</a>'.format(url))</p>
 
+  <p>
+    <a href="#" onclick="window.close(); return false;">$_("Cancel")</a> |
```

Comment:
This isn't necessary for this PR, but noting that we may want to move this into Webpack in the future.  This could be a good first issue, and the solution could be modeled on our recent cancel link changes (#9795).

Definitely not a blocker.